clients/lsp-credo.el

```diff
@@ -0,0 +1,99 @@
+;;; lsp-credo.el --- lsp-mode Credo integration -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2023 Wilhelm H Kirschbaum
+
+;; Author: Wilhelm H Kirschbaum
+;; Keywords: lsp, elixir, credo
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP Client for Elixir Credo
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-credo nil
+  "Settings for credo language server."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/elixir-tools/credo-language-server")
+  :package-version '(lsp-mode . "8.0.1"))
+
+(defcustom lsp-credo-command '("credo-language-server" "--stdio=true")
+  "The command that starts credo-language-server."
+  :type '(repeat :tag "List of string values" string)
+  :group 'lsp-credo
+  :package-version '(lsp-mode . "8.0.1"))
+
+(defcustom lsp-credo-version "0.1.1"
+  "Credo language server version to download.
+It has to be set before `lsp-credo.el' is loaded and it has to
+be available here: https://github.com/elixir-tools/credo-language-server/releases."
+  :type 'string
+  :group 'lsp-credo
+  :package-version '(lsp-mode . "8.0.1"))
+
+(defcustom lsp-credo-download-url
+  (format (concat "https://github.com/elixir-tools/credo-language-server"
```

Comment:
Will it be easier for you if i make sure the bin/credo-language-server script is always uploaded as an asset on the github release?

Reply:
Thanks, i updated the PR to use the new release file structure.